gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.7-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.0-all.zip
```

Comment:
You also need to check in the downloaded/updated versions of the following files:
```
gradle/wrapper/gradle-wrapper.jar
gradlew.bat
gradlew
```
Actually, the `gradle/wrapper/gradle-wrapper.properties` is probably out of sync too.